spid-testenv.py

```diff
@@ -562,10 +567,11 @@ def _load(self):
             self._save()
 
     def _save(self):
-        with open(self.FILE_NAME, 'w') as fp:
+        with open(self._filename, 'w') as fp:
             json.dump(self.users, fp, indent=4)
 
     def __init__(self, *args, **kwargs):
+        AbstractUserManager.__init__(self, *args, **kwargs)
```

Comment:
meglio usare `super`

Reply:
Volevo evitare di introdurre la dipendenza da Python 3, ma si può fare

Reply:
@alexrj Certo. Siamo comunque in caso in cui è più una questione stilistica (secondo me una chiamata con super alla maniera di python2 va benone per mantenere la compatibilità).

Domanda: python2 deve continuare ad essere supportato?

Reply:
Fatto. Python 2 secondo me possiamo supportarlo finché non ci costa troppo lavoro... finché siamo nel campo delle questioni stilistiche teniamolo. Cosa ne pensi?

Reply:
Sì, sono d'accordo. Una volta che sarà attiva la suite di test e la CI sarà più esplicito anche il supporto direi.